lib/pavilion/builder.py

```diff
@@ -319,7 +319,7 @@ def _update_src(self):
 
         if (src_url is not None
                 and ((src_download == 'missing' and found_src_path is None)
-                     or src_download == 'latest')):
```

Comment:
You've just changed the behavior of 'latest' such that it is now doing the same as 'missing'.  That's not the problem.  It seems that the files are getting downloaded, but they're not getting copied into their expected destination where Pavilion can find them.